data/human/campaign events.txt

```diff
@@ -155,11 +155,87 @@ mission "event: war begins"
+			choice
+				`	"Yes, that's clear."`
+					goto summary
+				`	"Sorry, I didn't know. You won't have any trouble from me."`
+					goto summary
+				`	"Yeah, right. No way are you going to seize my ship."`
+			
+			`	The liaison officer glowers at you, then turns military police officers in full combat gear materialize behind you, grabbing you by the arms and handcuffing you tightly. As you're frogmarched toward a waiting bunker, you watch as the <ship> lifts off without you, leaving you stranded here. In the bunker, one of the soldiers who arrested you frisks you, seizing your pilot's license and handing it to a sergeant who stamps the words "Permanently Revoked" across it. With those two words, you are barred from space forever. It has been less than a year since you bought your ship, and already you have lost the right to use it as the galaxy falls apart on you.`
+				die
+
+			label summary
+			`	"Glad to hear it," they reply, turning to calm a group of hysterical Paradise worlders.`
+			`	Re-entering the <ship>, you take off and head for orbit, narrowly missing the carrier squadron mentioned by the liaison officer. Only then do you see that you've left the video screen in your ship on, playing footage almost identical to what you saw on the ground not long ago. According to the newscaster, this debacle was all started yesterday, when a group of planets in the south of the galaxy sent the Republic Parliament an ultimatum: a refusal to pay taxes until reforms are made in the election of planetary representatives. This group, calling itself the "Free Worlds," has also asked for immediate reevaluation of all existing trade tariffs and regulations concerning pilot registration. Parliament outright refused to consider any of these demands.`
+			`	Then, only hours before you landed, nuclear explosions on Geminus and Martini destroyed the Republic Navy Yard and the Galactic Stock Exchange, crippling the military and economic center of the Republic. The "Free Worlds" disclaim any responsibility for the attacks, despite the timing and the fact that the attacks serve their purposes very well.`
+			`	Unless the terrorists can be found or turn themselves in, civil war is imminent. It has been less than a year since you bought your ship, and already the galaxy is falling apart on you.`
+				flee
+
+			label end
+			`	The liaison officer responds with a curt thank-you and turns around to calm a hysterical Paradise worlder. You re-enter the <ship>, and prepare to take off. It has been less than a year since you bought your ship, and already the galaxy is falling apart on you.`
```

Comment:
```suggestion
			`	The liaison officer responds with a curt thank-you and turns around to calm a hysterical Paradise Worlder. You re-enter the <ship>, and prepare to take off. It has been less than a year since you bought your ship, and already the galaxy is falling apart on you.`
```
Or
```suggestion
			`	The liaison officer responds with a curt thank-you and turns around to calm a hysterical Paradisian. You re-enter the <ship>, and prepare to take off. It has been less than a year since you bought your ship, and already the galaxy is falling apart on you.`
```